src/components/Search.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import "../styles/search.css";
+import getImages from "../requests/getImages";
+
+const Search = ({ setSearchResults }) => {
+  const [value, setValue] = useState();
+  const handleInputChange = (e) => setValue(e.target.value);
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    setSearchResults(await getImages(value));
+  };
+  return (
+    <>
```

Comment:
You could get away with removing the fragment tags here and just return the form